contrib/inventory/azure_rm.py

```diff
@@ -143,6 +148,10 @@
 If you don't need the powerstate, you can improve performance by turning off powerstate fetching:
 AZURE_INCLUDE_POWERSTATE=no
 
+Select hosts in Virtual machine scale set by setting below environment variable, 
```

Comment:
Virtual -> virtual, should be lowercase as it's not a title. Again I think "scaleset" should be one word.